.github/workflows/github_actions_update_npm_version.yml

```diff
@@ -0,0 +1,60 @@
+name: Updates npm package version and creates a PR
```

Comment:
I don't think this is up to date because @priya-xavier updated this file to be a dropdown select and not three variables